app/views/posts/show.html.erb

```diff
@@ -10,6 +10,17 @@
 	 </article>
```

Comment:
There is an error when the user goes to the post's author show page. Kindly see the screenshot below.
![post_error](https://user-images.githubusercontent.com/35504149/167018899-4815355b-bf8f-4733-ac1f-b2154b0003c2.png)

Reply:
**thank you @SajjadAhmad14 for the feedback, however, I've checked again and it's working on my side and it's not throwing any errors. I can add posts, comments, and likes and I can access the posts
here is the pages**

![image](https://user-images.githubusercontent.com/92228303/167020579-24aad56c-02ab-4aab-aa0b-f9d5c697ca52.png)
![image](https://user-images.githubusercontent.com/92228303/167020652-02ad105f-c4d5-4b90-8cec-15b4adbafc4c.png)
![image](https://user-images.githubusercontent.com/92228303/167020776-5f9c8002-ee8d-4233-8180-b06f56206551.png)